### PR TITLE
Update from Weblate.

### PR DESCRIPTION
**Pull request by xwikiorgci**

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-menu/xwiki-platform-menu-ui/src/main/resources/Menu/MenuTranslations.fr.xml [hunk 1]
```diff
@@ -77,7 +77,7 @@ menu.uix.scope.hint=Choix du contexte d'affichage du menu.
 
 # Menu Macro
 rendering.macro.menu.name=Menu
-### Missing: rendering.macro.menu.description=Displays a menu created using simple wiki syntax (nested lists and links).
+rendering.macro.menu.description=Affiche un menu créé en utilisant la syntaxe wiki simple (listes imbriquées et liens)
```
Comment (vmassol) [inline xwiki-platform-core/xwiki-platform-menu/xwiki-platform-menu-ui/src/main/resources/Menu/MenuTranslations.fr.xml]: Better: "en utilisant une syntax wiki simple". There's no such thing as "la syntaxe wiki simple" ;)

Comment (tmortagne) [reply xwiki-platform-core/xwiki-platform-menu/xwiki-platform-menu-ui/src/main/resources/Menu/MenuTranslations.fr.xml]: JSDennebouy does not seems to have a github account and won't see this comment so better directly fix it on https://l10n.xwiki.org.

Comment (atallahade) [reply xwiki-platform-core/xwiki-platform-menu/xwiki-platform-menu-ui/src/main/resources/Menu/MenuTranslations.fr.xml]: @vmassol Feel free to make the change here: https://l10n.xwiki.org/translate/xwiki-platform/menu-application/fr/?checksum=d605bf29454b7e9b

Comment (js-ddev) [reply xwiki-platform-core/xwiki-platform-menu/xwiki-platform-menu-ui/src/main/resources/Menu/MenuTranslations.fr.xml]: > JSDennebouy does not seems to have a github account and won't see this comment so better directly fix it on https://l10n.xwiki.org.

Yes I do !
Sorry Vincent, there may be some errors on technical translations.
The main modifications are changes of "mail" and "email" to "courriel".

Comment (tmortagne) [reply xwiki-platform-core/xwiki-platform-menu/xwiki-platform-menu-ui/src/main/resources/Menu/MenuTranslations.fr.xml]: OK I see you associated the commits with your account, cool :)

**Pull request merged**